openlcs/libs/driver.py

```diff
@@ -134,6 +134,8 @@ def get_paginated_data(self, url, query_params=None):
                 data = response.json()
                 yield from data["results"]
                 url = data.get("next")
+                if query_params:
+                    query_params = None
```

Comment:
Why use a different setting compared with https://github.com/RedHatProductSecurity/openlcs/blob/94c7cbb03af303ba902ee52018c917edacc0a2d7/openlcs/libs/corgi.py#L376

Reply:
What do you mean? I'm not sure I understand.

Reply:
One set `query_params = None ` directly, and another one need check if `query_params` is True

Reply:
I think both works and has no side effect. anyway it's good to keep it consistent.

I'll change `corgi.py` to be consistent with here.